src/Components/Publishing/ToolTip/GeneToolTip.tsx

```diff
@@ -36,6 +36,9 @@ export const GeneToolTip: React.SFC<GeneProps> = (props, context) => {
 
 const Wrapper = styled.div`
   width: 240px;
+  a:hover {
```

Comment:
Tried putting it in the `styled.a` just below but it doesn't seem to take, I think this may be because of some styles in Force that are overriding it.

Reply:
Got it ok makes sense, I was just saying because we lose a bit of context when having this selector nested inside of `Wrapper`